lib/src/wolt_modal_sheet.dart

```diff
@@ -164,49 +106,238 @@ class WoltModalSheet<T> extends StatefulWidget {
       ),
     );
   }
+
+  /// The state from the closest instance of this class that encloses the given
+  /// context.
+  ///
+  /// Typical usage is as follows:
+  ///
+  /// ```dart
+  /// WoltModalSheet.of(context).popPage()
+  /// ```
+  ///
+  /// If there is no [WoltModalSheet] in the give `context`, this function will throw
+  /// a [FlutterError] in debug mode, and an exception in release mode.
+  ///
+  /// This method can be expensive (it walks the element tree).
+  static WoltModalSheetState of(BuildContext context) {
+    // Handles the case where the input context is a navigator element.
+    WoltModalSheetState? wms;
```

Comment:
This isn't public but i suggest to use words as they're readable and searchable if needed. 
```suggestion
    WoltModalSheetState? woltModalSheetState;
```